llarp/util/aligned.hpp

```diff
@@ -28,14 +28,9 @@ namespace llarp
 {
   /// aligned buffer that is sz bytes long and aligns to the nearest Alignment
   template <size_t sz>
-#ifdef _WIN32
-  // We CANNOT align on a 128-bit boundary, malloc(3C) on win32
-  // only hands out 64-bit aligned pointers
-  struct alignas(uint64_t) AlignedBuffer
-#else
   struct alignas(std::max_align_t) AlignedBuffer
-#endif
   {
+    static_assert(alignof(std::max_align_t) <= 16, "insane alignment");
```

Comment:
perhaps a slightly more descriptive message would be good here.  Granted this is compile-time and will point to this source line (and the context around it), but still...